src/data/pokemon/egg_moves.h

```diff
@@ -1,12 +1,11 @@
 #include "constants/moves.h"
 
-#define EGG_MOVES_SPECIES_OFFSET 20000
-#define EGG_MOVES_TERMINATOR 0xFFFF
-#define egg_moves(species, moves...) (SPECIES_##species + EGG_MOVES_SPECIES_OFFSET), moves
+static const u16 sNoneEggMoveLearnset[] = {
+    MOVE_UNAVAILABLE,
+};
 
-const u16 gEggMoves[] = {
 #if P_FAMILY_BULBASAUR
-    egg_moves(BULBASAUR,
+static const u16 sBulbasaurEggMoveLearnset[] = {
         MOVE_SKULL_BASH,
         MOVE_CHARM,
         MOVE_PETAL_DANCE,
```

Comment:
Could the script handle the formatting as well?

Reply:
What do you mean with formatting?

Reply:
it has 2 tabs instead of one like usually for movesets

Reply:
Ah, that's a fair relic and a trivial fix, yeah, thanks!